pkg/maps/ipcache/ipcache.go

```diff
@@ -143,8 +144,8 @@ type RemoteEndpointInfo struct {
 }
 
 func (v *RemoteEndpointInfo) String() string {
-	return fmt.Sprintf("identity=%d encryptkey=%d tunnelendpoint=%s nodeid=%d",
-		v.SecurityIdentity, v.Key, v.TunnelEndpoint, v.NodeID)
+	return fmt.Sprintf("identity=%d encryptkey=%d tunnelendpoint=%s nodeid=0x%s",
+		v.SecurityIdentity, v.Key, v.TunnelEndpoint, strconv.FormatUint(uint64(v.NodeID), 16))
```

Comment:
Do we also want to update https://github.com/cilium/cilium/blob/c2c5ed3334ae8150fd06080c2a937ec9f3041964/cilium/cmd/bpf_nodeid_list.go#L84 and https://github.com/cilium/cilium/blob/c2c5ed3334ae8150fd06080c2a937ec9f3041964/cilium/cmd/node_id_list.go#L54?

Reply:
Yep, good suggestion.